transformer_engine/common/common.h

```diff
@@ -261,12 +289,20 @@ struct QuantizationConfig {
   NVTETensor noop_tensor = nullptr;
   Float8BlockScaleTensorFormat float8_block_scale_tensor_format =
       Float8BlockScaleTensorFormat::GEMM_READY;
+  size_t rng_seed = 0;
+  size_t rng_sequence = 0;
```

Comment:
This is not compatible with cuda graphs and will need to be changed to NVTETensor.

Reply:
I don't think this is doable within deadlines. We should defer to 2.9. We shouldn't make any API-stability commitments for FP4-related features in this release.

Reply:
It was fixed in the latest version of the pr, resolving.